bucketlist/auth.py

```diff
@@ -0,0 +1,42 @@
+from flask_httpauth import HTTPBasicAuth, HTTPTokenAuth
+from itsdangerous import TimedJSONWebSignatureSerializer as Serializer
+from flask import g
+from bucketlist import app
+from bucketlist.models import User
+
+
+auth = HTTPBasicAuth()
+auth_token = HTTPTokenAuth("Token")
+
+
+def verify_password(username, password):
+    """
+    Login verification
+    """
+    g.user = User.query.filter_by(username=username).first()
+    if g.user is None:
+        return False
+    elif g.user.check_password(password):
+        return g.user
+
```

Comment:
The condition in if is not related to the condition in elif. Might as well be 2 diff ifs. But also `pythonic` way people prefer `if not g.user` rather than `if g.user is None`